bindings/js/api_test.js

```diff
@@ -0,0 +1,118 @@
+var path = require('path');
+var HaclWasm = require(path.resolve(__dirname,'./api.js'))
+var test_vectors = require(path.resolve(__dirname, './api.json'));
+var eq_array = function(a1, a2) {
+  if (a1.length !== a2.length) {
+    return false
```

Comment:
semicolons at the end of the line would be great for consistency with the rest of your code... have you thought about running `jslint` somewhere?

Reply:
Indeed, I used `jshint` to get consistency now